recipe/meta.yaml

```diff
@@ -12,30 +12,21 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: 1
+  # These are present when the new environment is created
+  # so we have to exempt them from the list of initial files
+  # for conda-build to realize they should be included.
   entry_points:
     - conda = conda.cli:main
     - conda-env = conda_env.cli.main:main
   always_include_files:
     - bin/conda                            # [unix]
     - bin/activate                         # [unix]
     - bin/deactivate                       # [unix]
-    - etc/fish/conf.d/conda.fish           # [unix]
     - Scripts/activate.bat                 # [win]
     - Scripts/activate                     # [win]
-    - Scripts/conda.exe                    # [win]
-    - Scripts/conda-script.py              # [win]
-    - Scripts/conda-script.pyc             # [win]
```

Comment:
Do you know if we still need to keep these things, @kalefranz? They seem to be in the final `conda` package. Not sure `conda` is adding these to the environment. Also we will need to build this with `conda` 4.2.13 in the environment if that is relevant.